### ISSUE TITLE
Diagnostics fail in new files

### ISSUE THREAD

**Issue by alexander-js** [elm-tooling/elm-language-server#515]
Newly created files get `Cannot read property 'tree' of undefined` from the LS when trying to run diagnostics.

```
[Info  - 15:24:51] Document Symbols were requested
[Trace - 15:24:51] Received response 'textDocument/documentSymbol - (40)' in 1ms. Request failed: Request textDocument/documentSymbol failed with message: Cannot read property 'tree' of undefined (-32603).
[Error - 15:24:51] Request textDocument/documentSymbol failed.
  Message: Request textDocument/documentSymbol failed with message: Cannot read property 'tree' of undefined
  Code: -32603 
```
## Expected Behavior

I expect the diagnostics to work as per usual.

## Current Behavior

Newly created files fail when attempting diagnostics.
As a temporary fix, restarting the LS makes diagnostics work in the file.

## Steps to Reproduce (for bugs)

1. Clone https://github.com/rtfeldman/elm-spa-example
2. Create `src/Whatever.elm`
3. Edit the file
4. Save the file

## Context

Related settings in VSCode's `settings.json`:
```
{
    "elmLS.trace.server": "verbose",
    "elmLS.onlyUpdateDiagnosticsOnSave": true,

    "[elm]": {
        "editor.tabSize": 4,
        "editor.detectIndentation": false,
        "editor.codeLens": false
    }
}
```

## Your Environment

* Version used: 2.0.3
* Editor name and version (e.g. VSCode 1.36.1): VSCode 1.53.1
* Environment name and version (e.g. node.js 5.4): 12.16.2
* Operating System and version: Ubuntu 20.04.1

**Comment by jmbockhorst** [comment]
Regression from 46fb16ed58631de47987c010dfad6fb8067aada1.

**Comment by Zimmi48** [comment]
Hi! I've just hit this bug with elm-language-server 2.0.3 (the version currently available in nixpkgs). This isn't surprising given that it was fixed in 2.1. But then I updated the version available in nixpkgs to 2.2 and installed it and I'm hitting the bug again. Was the regression reintroduced somehow? (I didn't manage to install and test version 2.1 to confirm that it works.)

**Comment by jmbockhorst** [comment]
I just tested this on 2.2 and it works as expected. You could be running into something else.
